server/memphis_handlers_configurations.go

```diff
@@ -124,6 +125,47 @@ func changePMRetention(pmRetention int) error {
 	if err != nil {
 		return err
 	}
+	var stations []models.Station
```

Comment:
only one broker should handle this change, remove the message sent to the brokers + the listener for such updates + global var